assets/stylesheets/kitten/helpers/typography/_font-scale.scss

```diff
@@ -0,0 +1,43 @@
+/// Apply a font scale using according to media view.
+///
+/// Font-step will be computed directly using the modular scale. It is optional
+/// and set to 0 by default the regular font step.
+///
+/// Font-step-on-media is optional. If it is not defined, it will be set to the
+/// regular font step.
+///
+/// media-size will be computed directly using the modular scale. It is
+/// optional and is set to media 's' by default. It's the minimum media size
+/// from where the large font step has to be applied.
+///
+/// @group typography
+///
+/// @parameter {Number} $font-step                 - the modular scale step to compute
+/// @parameter {Number} $font-step-on-media [null] - the font step on large media
```

Comment:
Et ici, comme pour le commentaire précédent, avoir `the font step on the media specified by $media-size` ?